examples/docker/census-on-omnisci/census-omnisci.py

```diff
@@ -143,20 +245,27 @@ def measure(name, func, *args, **kw):
     t0 = time.time()
     res = func(*args, **kw)
     t1 = time.time()
-    print(f'{name}: {t1 - t0} sec')
+    print(f"{name}: {t1 - t0} sec")
     return res
 
 
 def main():
+    if len(sys.argv) != 2:
+        print(
+            f"USAGE: docker run --rm -v /path/to/dataset:/dataset census-omnisci <data file name in /path/to/dataset>"
```

Comment:
It seemed to me that in modin it is customary to designate such places filled by the user with `[]` symbols. What do you think?
```suggestion
            f"USAGE: docker run --rm -v /path/to/dataset:/dataset census-omnisci [data file name in /path/to/dataset]"
```

Reply:
I think square brackets are usually used for optional arguments. This argument is not optional.